geoportal/geoportailv3_geoportal/views/fulltextsearch.py

```diff
@@ -445,23 +445,23 @@ def featuresearch(self, layers=None):
                     return []
 
                 try:
-                    esricoll = geojson.loads(content)
+                    log.info(fiona.__version__)
+                    log.info(fiona.get_gdal_release_name())
+                    log.info(fiona.supported_drivers)
+                    mf = fiona.MemoryFile(content)
+                    esricoll = mf.open(driver='ESRIJSON')
```

Comment:
shoudn't it be closed ? (or use with a context manager)

Reply:
yes indeed.